backend/DB/api/routes.py

```diff
@@ -0,0 +1,239 @@
+from backend.DB.api import app, db, queries
+from flask import jsonify, request, abort, render_template
+
+from backend.DB.api.tables import User, Pair, Reward, Threshold
+
+
+@app.route('/')
+def graph_display():
+    return render_template('graph_display.html')
+
+
+def format_users(users):
+    output = []
+    for user in users:
+        output.append(
+            {'username': user.username, 'name': user.name, 'image': user.image, 'active': user.active})
+    return jsonify(output)
+
+
+@app.route('/status/data')
+def status_data():
+    return jsonify({"cake_count": len(queries.get_pair_since_last_reward("cake")),
+                     "pizza_count": len(queries.get_pair_since_last_reward("pizza")),
+                     "pizza_thres": queries.get_threshold("pizza").threshold,
+                     "cake_thres": queries.get_threshold("cake").threshold,
+                     "unused_cake": queries.get_unused_rewards_count_by_type("cake"),
+                     "unused_pizza": queries.get_unused_rewards_count_by_type("pizza"),
+                     "last_pair": ["esog", "mleik"]})
+
+
+@app.route('/api/user/add', methods=['POST'])
```

Comment:
Maybe the path for this route should be `/users/add` rather than `/api/user/add`